test/optimal_filter_validation.yaml

```diff
@@ -11,4 +11,4 @@ model:
     obs_noise_std: 0.01
     sigma: 1.0
     lambda: 1.0
-    nu: 2.5
+    nu: 0.5
```

Comment:
This is what I was referring to with

> to be consistent with the Matern covariance function being used (specifically using the simplified form of the Matern covariance for ν=0.5 as the test case)

As the Matern covariance function is particularly simple for `nu=0.5` I changed to using this value so that in the `optimal_filter_validation.jl` definition of `R`
https://github.com/Team-RADDISH/ParticleDA.jl/blob/1f3e8e3ba331d4af00e2b802eb009489e5ff79ef/test/optimal_filter_validation.jl#L99-L105
we can just use a simple exponential expression rather than having to use `SpecialFunctions` to load the `gamma` and `besselk` functions. It might still be better to do this though I guess from a robustness standpoint so that the test is valid for any value of `nu`. Or at least I should perhaps put a comment in the YAML file to make clear this value shouldn't be changed?

Reply:
> we can just use a simple exponential expression rather than having to use `SpecialFunctions` to load the `gamma` and `besselk` functions.

If the worry is to load another package during the tests, `SpecialFunctions` is already pulled in by `GaussianRandomFields`, so it isn't much more work.

> It might still be better to do this though I guess from a robustness standpoint so that the test is valid for any value of `nu`.

I think that'd be better, yes.

> Or at least I should perhaps put a comment in the YAML file to make clear this value shouldn't be changed?

At very least this, yes, please.